sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/query.sql

```diff
@@ -0,0 +1,1461 @@
+          THEN client_id
+      END
+    ) AS history_search_term_group_open_tab_users
+--search_term_group_remove_all
+    ,
+    SUM(history_search_term_group_remove_all) AS history_search_term_group_remove_all,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN history_search_term_group_remove_all > 0
+          THEN client_id
+      END
+    ) AS history_search_term_group_remove_all_users
+--search_term_group_remove_tab
+    ,
+    SUM(history_search_term_group_remove_tab) AS history_search_term_group_remove_tab,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN history_search_term_group_remove_tab > 0
+          THEN client_id
+      END
+    ) AS history_search_term_group_remove_tab_users
+--search_term_group_tapped
+    ,
+    SUM(history_search_term_group_tapped) AS history_search_term_group_tapped,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN history_search_term_group_tapped > 0
+          THEN client_id
+      END
+    ) AS history_search_term_group_tapped_users
+--shared
+    ,
+    SUM(history_shared) AS history_shared,
+    COUNT(DISTINCT CASE WHEN history_shared > 0 THEN client_id END) AS history_shared_users
+/*FxA*/
+--sync_failed
+    ,
+    SUM(sync_failed) AS sync_failed,
+    COUNT(DISTINCT CASE WHEN sync_failed > 0 THEN client_id END) AS sync_failed_users
+--sync_account_opened
+    ,
+    SUM(sync_account_opened) AS sync_account_opened,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN sync_account_opened > 0
+          THEN client_id
+      END
+    ) AS sync_account_opened_users
+--sync_account_send_tab
+    ,
+    SUM(sync_account_send_tab) AS sync_account_send_tab,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN sync_account_send_tab > 0
+          THEN client_id
+      END
+    ) AS sync_account_send_tab_users
+--sync_account_sign_in_to_send_tab
+    ,
+    SUM(sync_account_sign_in_to_send_tab) AS sync_account_sign_in_to_send_tab,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN sync_account_sign_in_to_send_tab > 0
+          THEN client_id
+      END
+    ) AS sync_account_sign_in_to_send_tab_users
+--sync_account_sync_now
+    ,
+    SUM(sync_account_sync_now) AS sync_account_sync_now,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN sync_account_sync_now > 0
+          THEN client_id
+      END
+    ) AS sync_account_sync_now_users
+--sync_auth_closed
+    ,
+    SUM(sync_auth_closed) AS sync_auth_closed,
+    COUNT(DISTINCT CASE WHEN sync_auth_closed > 0 THEN client_id END) AS sync_auth_closed_users
+--sync_auth_opened
+    ,
+    SUM(sync_auth_opened) AS sync_auth_opened,
+    COUNT(DISTINCT CASE WHEN sync_auth_opened > 0 THEN client_id END) AS sync_auth_opened_users
+--sync_auth_other_external
+    ,
+    SUM(sync_auth_other_external) AS sync_auth_other_external,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN sync_auth_other_external > 0
+          THEN client_id
+      END
+    ) AS sync_auth_other_external_users
+--sync_auth_paired
+    ,
```

Comment:
weird comma, seems like there's a bunch of these around here. Could you please move it to be located immediately after field name?